lib/chef/provisioning/aws_driver/super_lwrp.rb

```diff
@@ -33,6 +33,12 @@ def self.attribute(attr_name, validation_opts={})
             super
           end
         end
+
+        # FUUUUUU cloning
```

Comment:
Yep.  Not dangerous though.  All Cheffish and chef-provisioning objects have been doing this for a very long time, no issues yet.

The reason for it is, cloning kills patchiness.  We need to be able to declare multiple things with their own properties without polluting them with other resources' properties.